README.md

```diff
@@ -1,10 +1,10 @@
 # Release Information
 
-- **Version**:  1.0.1-1006
+- **Version**:  1.1.0-1003
```

Comment:
Please make this version consistent with other SPs

Reply:
This is how it has always been for TIM.